packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue

```diff
@@ -43,6 +43,7 @@ const emit = defineEmits(['update:selection']);
 
 <style scoped>
 .p-dropdown {
+	max-width: 20rem;
```

Comment:
We might also truncate the names of output with an ellipsis if they are too big.
@ngraham76 we should look on improving those output dropdown to be more complex, with multiline information containing Name, status, createdOn/updatedOn time, if it's a saved asset or not.